client/src/components/DataFiles/DataFilesListing/DataFilesListing.js

```diff
@@ -28,6 +28,16 @@ const DataFilesListing = ({ api, scheme, system, path, isPublic }) => {
     }),
     shallowEqual
   );
+  const systems = useSelector(
+    state => state.systems.storage.configuration,
+    shallowEqual
+  );
+  const sharedWorkspaces = systems.find(e => e.scheme === 'projects');
+  const systemPrefix = useSelector(
+    state => state.workbench.config.systemPrefix
+  );
+  const isPortalProject = system && system.startsWith(systemPrefix);
```

Comment:
A simpler way to check whether or not you're looking at project systems might be to check the `scheme` being used